pkg/stanza/go.mod

```diff
@@ -13,45 +13,242 @@ require (
 	github.com/stretchr/testify v1.8.0
 	go.opentelemetry.io/collector v0.61.0
 	go.uber.org/zap v1.23.0
-	golang.org/x/exp v0.0.0-20200331195152-e8c3332aa8e5 // indirect
-	golang.org/x/sys v0.0.0-20220808155132-1c4a2a72c664
+	golang.org/x/sys v0.0.0-20220829200755-d48e67d00261
 	golang.org/x/text v0.3.7
 	gonum.org/v1/gonum v0.12.0
 	gopkg.in/yaml.v2 v2.4.0
 )
 
 require (
+	github.com/go-kit/log v0.2.1
+	github.com/grafana/loki v1.6.2-0.20220927161239-f3b72c8b591e
 	github.com/influxdata/go-syslog/v3 v3.0.1-0.20210608084020-ac565dc76ba6
 	github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage v0.61.0
+	github.com/prometheus/client_golang v1.13.0
+	github.com/prometheus/common v0.37.0
+	github.com/prometheus/prometheus v1.8.2-0.20211119115433-692a54649ed7
```

Comment:
In the long run, I'd like to reduce the scope of the `pkg/stanza` codebase by moving receiver-specific input operators into the appropriate receiver modules. With that in mind, can I ask that all the promtail related code reside in the new receiver codebase? You should be able to implement and register an operator with exported functions.

Reply:
moved promtail related code to receiver/promtailreceiver directory